xbmc/pvr/PVRDatabase.cpp

```diff
@@ -488,36 +490,17 @@ int CPVRDatabase::Get(CPVRChannelGroup &group)
 
 bool CPVRDatabase::PersistChannels(CPVRChannelGroup &group)
 {
-  bool bReturn(true);
+  for (auto &member : group.m_members) {
```

Comment:
Yeah this is CLion outsmarting me, I haven't configured the correct code style in it yet. Will update.